modules/avatar.js

```diff
@@ -55,20 +57,23 @@ async function targetImg(int, size = 256) {
   }
   const target = (int.options[int.guild ? "getMember" : "getUser"]('user')) ?? int.user;
   // @ts-ignore
-  return target.displayAvatarURL({ format: 'png', size, dynamic: true });
+  return target.displayAvatarURL({ extension: 'png', size, dynamic: true });
 }
 
 /**
  * Apply a filter function with parameters. Useful for when there isn't much logic to it
  * @param {discord.CommandInteraction} int
  * @param {string} filter filter to apply
  * @param {any[]?} params array of params to pass into the filter function
+ * @param {Boolean} both
  */
-async function basicFilter(int, img, filter, params) {
+async function basicFilter(int, img, filter, params, both) {
   if (params) img[filter](...params);
   else img[filter]();
   const output = await img.getBufferAsync(Jimp.MIME_PNG);
-  return int.editReply({ files: [output] });
+  const image = new discord.AttachmentBuilder(output, { name: "image.png" });
+  const embed = both ? u.embed().setTitle(filter).setImage(`attachment://${image.name}`).setFooter({ text: "you provided both a user and a file, so I defaulted to using the file" }) : u.embed().setTitle(filter).setImage(`attachment://${image.name}`);
```

Comment:
No need to rewrite the whole embed. You can do something like

```suggestion
  const embed = u.embed().setTitle(filter).setImage(`attachment://${image.name}`).setFooter(both ? { text: "you provided both a user and a file, so I defaulted to using the file" } : null) ;
```